### PR TITLE
Add note 

### PR DESCRIPTION
**Pull request by vermasavita**
I'll use commented code later. Please ignore that for now.

### CODE REVIEW

**Review by surbhi84 — APPROVED** [review]
- Clean UI looks great.
- Liked that you're using localStorage.
- Code looks mergeable to me.

Suggestions :
- Read about absolute paths your imports will look much cleaner.
- Instead of directly putting svg's in your code you can make use of react icons.

File: src/components/create-note/CreateNote.jsx [hunk 1]
```diff
@@ -1,21 +1,43 @@
...
             <input
               type="text"
-              id="text"
+              id="title"
               name="title"
               placeholder="Title"
-              autocomplete="off"
+              value={note.title}
+              onChange={(event) =>
+                setNote(() => ({ ...note, title: event.target.value }))
```
Comment (sashaboi) [inline src/components/create-note/CreateNote.jsx:37]: A user is allowed to create an empty note, maybe in future iterations you may want to check if note is empty and not allow that.

**Review by sashaboi — APPROVED** [review]
everything looks good! 
empty note creation should be handled
homepage button could be styled better for on hover
github button link could be updated on the homepage.

looking forward to rest of the project.

**Comment by kumsomi** [comment]
- Code is clean.

- UI is looking good. Except for some design change at hover.

File: src/components/create-note/CreateNote.jsx (hunk 1 above)
Comment (vermasavita) [reply src/components/create-note/CreateNote.jsx:37]: Yes, that's in mind.

**Pull request merged**